### DISCUSSION TITLE
Importing External Files

### DISCUSSION THREAD

**Discussion by anatoli-dp** [GrapesJS/grapesjs#3756] [Q&A]
Ok so i know this has been asked before but as it has been a while ago i am unsure if anything has changed since then. what is considered the proper way to add styles and javascript files to the editor without including them in the startup config. also, for javascript files, how would i ensure the script stays at the bottom of the structure and doesnt end up somewhere in the middle of the page as i continue editing the page. sorry if it not make sense but if u need clarification just ask

**Comment by artf** [comment]
It should be all mentioned here https://grapesjs.com/docs/modules/Components-js.html
When you define component-related scripts, you decide where to append them.
